src/Switch/index.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { forwardRef } from 'react';
+import styled from 'styled-components';
+import { CustomCheckboxContainer, CustomCheckboxInput } from '@reach/checkbox';
+import { MinervaProps, Box, systemProps } from '../layout';
+import VisuallyHidden from '../VisuallyHidden';
+
+const switchSizes = {
+  lg: {
+    width: '55px',
+    height: '25px',
+  },
+  md: {
+    width: '40px',
+    height: '18px',
+  },
+  sm: {
+    width: '33px',
+    height: '14.5px',
+  },
+};
+
+const SwitchContainer = styled(Box)<SwitchProps>`
+  position: relative;
+  width: ${props =>
+    props.switchSize
+      ? switchSizes[props.switchSize].width
+      : switchSizes['md'].width};
+  display: inline-block;
+  vertical-align: middle;
+  text-align: left;
+  ${systemProps}
+`;
+
+const ControlBox = styled(Box)<SwitchProps>((props: SwitchProps) => ({
+  display: 'block',
+  overflow: 'hidden',
+  cursor: 'pointer',
+  border: '0 solid #ccc',
+  borderRadius: '20px',
+  margin: '0',
+  height: props.switchSize
+    ? switchSizes[props.switchSize].height
+    : switchSizes['md'].height,
+
+  ':focus-within': {
+    borderColor: '#a4cafe',
+    boxShadow: '0 0 0 3px rgba(118,169,250,.45)',
+    outline: 0,
+  },
+}));
+
+const SwitchInner = styled.span<CustomSwitchProps>`
+  display: block;
+  width: 200%;
```

Comment:
I see that this is what's making the background color, but using a percentage for something like this seems a bit confusing. I'm likely missing something, but would it be possible to just change the background color here instead of using this combined with `:before` and `:after`?